modules/ROOT/pages/testing.adoc

```diff
@@ -9,10 +9,4 @@ between the Module and Mule is required to ensure compatibility with the Mule ec
 Here are a series of articles that explain in detail how to test your module. It provides a step-by-step introduction on how to test a module. It explains the basis of a Module Test Case and how to execute a simple operation from a Module.
 
 * <<testing-writing-your-first-test-case#, Writing your first test case>>
-* xref:testing-writing-your-first-test-case.adoc#testing-flowrunner[FlowRunner]
-* Operations, Scopes and Routers
-* Sources
-* DataSense Resolution
-* ValueProviders Resolution
-* Handling ClassLoading Issues
-* Best Practices
+* xref:testing-writing-your-first-test-case.adoc#testing-flowrunner[FlowRunner]
```

Comment:
This isn't actually an addition, it's in the doc now. Leaving as a bullet list of one item is not a good practice, and likely some set of customers finds it useful to have it called out.

Reply:
I believe the devs intended to write something on those topics in bullets that were removed, so checking in with them if they need a JIra for that would be a good idea. They've prob forgotten about it.

Reply:
I just reminded them yesterday, it's all in the bug.